### PR TITLE
Publishing cleanup

### PR DESCRIPTION
**Pull request by alloy**

### CODE REVIEW

File: change/monorepo-scripts-3c5e63c1-709c-4d5d-af40-308b1d3df4ce.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Literally none",
+  "packageName": "monorepo-scripts",
+  "email": "eloy.de.enige@gmail.com",
+  "dependentChangeType": "none"
+}
```
Comment (alloy) [inline change/monorepo-scripts-3c5e63c1-709c-4d5d-af40-308b1d3df4ce.json:7]: @kenotron It keeps thinking I made changes to this, any idea why?

Comment (kenotron) [reply change/monorepo-scripts-3c5e63c1-709c-4d5d-af40-308b1d3df4ce.json:7]: Flip that project to "private: true" perhaps would make it stop :)

File: scripts/just.config.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export const types = tscTask({ emitDeclarationOnly: true });
 
 export const build = () =>
   esbuildTask({
-    entryPoints: glob.sync("src/**/*.ts"),
+    entryPoints: glob.sync("src/**/*.{ts,tsx}"),
```
Comment (kenotron) [inline scripts/just.config.ts:9]: you technically did make changes in /scripts which has a package.json in the "workspace" - flip that package.json to have private flag would stop that noise.

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "test": "lage test",
     "lint": "lage lint",
     "lage": "lage",
+    "ci": "yarn build && yarn test && yarn lint && yarn checkchange",
```
Comment (kenotron) [inline package.json:17]: nice - I think all you need to do this: `yarn lage build test lint && yarn checkchange`

**Review by kenotron — COMMENTED** [review]
This is good enough as-is, but I have some good suggestions that you should probably consider.

File: scripts/just.config.ts (hunk 2 above)
Comment (alloy) [reply scripts/just.config.ts:9]: Yes, very true, my bad. I feel like I keep having to do it for this package but raised it at the wrong time. Earlier on there was a mode change from 644 to 755 iirc.

File: package.json (hunk 3 above)
Comment (alloy) [reply package.json:17]: :boom: Went from 4 sec to 3 sec

File: scripts/just.config.ts (hunk 2 above)
Comment (alloy) [reply scripts/just.config.ts:9]: Added `private: true` 👍

**Pull request merged**